curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/633928c6b0baf16c0b0d823b.md

```diff
@@ -0,0 +1,96 @@
+---
+id: 633928c6b0baf16c0b0d823b
+title: Step 41
+challengeType: 0
+dashedName: step-41
+---
+
+# --description--
+
+You have created two radio buttons. However, if you submit the form, it does not require you to check any of them.
+
+This is not good, since you need to know what type of account the user requires.
+
+To correct this add a `checked` attribute to `personal-account or business-account`.
+
+Also add a `legend` element with the text `Account type (required)` inside the `fieldset`.
+
+Remove the `Account` in the `label` tag. The `legend` element will let the user know what is `Personal` and `Business`.
+
+# --hints--
+
+You should use the `checked` attribute for the `input` tag.
+
+```js
+assert.equal(document.getElementsByName("account-type").required, true);
+```
```

Comment:
this still checks for required, not checked.

There is no tests for the `legend` element